pytest_dbfixtures/executors/postgresql.py

```diff
@@ -33,7 +33,8 @@ class PostgreSQLExecutor(TCPExecutor):
     """
 
     BASE_PROC_START_COMMAND = """{pg_ctl} start -D {datadir}
-    -o "-F -p {port} -c %s='{unixsocketdir}'" -l {logfile} {startparams}"""
+    -o "-F -p {port} -c log_destination='stderr' -c %s='{unixsocketdir}'"
```

Comment:
From documentation:
> The default is to log to stderr only.

What if someone wants to use *stderr*, *csvlog* or *syslog*. I think it should be a param in __init__ method.